app/src/master/res/values/setup.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
-    <string name="queue_id" tools:ignore="ExtraTranslation">55e5ca69-04a9-453a-88c1-73b13e83a227</string>
+    <string name="default_queue_id" tools:ignore="ExtraTranslation">55e5ca69-04a9-453a-88c1-73b13e83a227</string>
```

Comment:
I don't think there is a point in leaving the default queue ID. This is open source project and it might be confusing why there is a default queue ID that is site-specific and won't work with just any site.

Reply:
https://glia.atlassian.net/browse/MUIC-667